README.md

```diff
@@ -7,76 +7,85 @@
 > This repository does not pretend to give you all answers for LinkedIn questions, rather it's a starting guide to help you prepare for the LinkedIn skills quiz and to know what to expect in the main exam or which areas to target in your exam preparations. Should you have found some incorrect answers, or want to contribute your answers, please feel free to create any PR changes - all are welcome!
 
 :point_right: If you want to improve your skills on another test platform [**Pluralsight IQ Skill**](https://www.pluralsight.com/product/skill-iq)
-find **questions and answers** in **@Timo Körner** [repository](https://github.com/tik9/pluralsight-skill-test).
+find **questions and answers** in **@Timo Körner** [repository](https://github.com/tik9/pluralsight-skill-test):
 
-| Pluralsight IQ Skill quiz                                                                                                | Status      |
-| ------------------------------------------------------------------------------------------------------------------------ | ----------- |
-| [Angular](https://github.com/tik9/pluralsight/blob/master/angular.md)                                                    | ok          |
-| [Aws Big Data](https://github.com/tik9/pluralsight/blob/master/aws_big_data.md)                                          | ok          |
-| [Aws Machine Learning](https://github.com/tik9/pluralsight/blob/master/aws_machine_learning.md)                          | ok          |
-| [Building Websites with HTML, CSS, and JavaScript](https://github.com/tik9/pluralsight/blob/master/building-websites.md) | ok          |
-| [Java](https://github.com/tik9/pluralsight/blob/master/java.md)                                                          | ok          |
-| [Mysql](https://github.com/tik9/pluralsight/blob/master/mysql.md)                                                        | ok          |
-| [Powershell](https://github.com/tik9/pluralsight/blob/master/powershell.md)                                              | non-offical |
-| [Python](https://github.com/tik9/pluralsight/blob/master/python_core.md)                                                 | ok          |
+- [Angular](angular.md)
+- [AngularJs](angularjs.md)
+- [Aws Big Data](aws_big_data.md)
+- [Aws Machine Learning](aws_machine_learning.md)
+- [Building Websites](building_websites.md)
+- [Building Web Apps with React](building_web_apps_with_react.md)
+- [Java](java.md)
+- [Javascript](javascript.md)
+- [Mysql](mysql.md)
+- [Powershell](powershell.md)
+- [Python Core](python_core.md)
+- [Python Data Analysis](python_data_analysis.md)
+- [Python applied Data Mining](python_applied_data_mining.md)
+- [Python Web Scraping](python_web_scraping.md)
+- [Python Exploratory Data Analysis](python_exploratory_data_analysis.md)
+- [Python Clean Data](python_clean_data.md)
+- [Python Data Visualization](python_data_visualization.md)
```

Comment:
I know your original Readme has the same style(without table) but could you please revert the table back(please), I want to let other people know some more info before they redirected to your repository. For example, does it contains answers or not? What do you think? and additionally, I would like to keep a similar style for both tables in my Readme. In my opinion, it looks more beautiful. Thank you for understanding

Reply:
ok did that.